include/effects/matrix/spectrumeffects.h

```diff
@@ -189,16 +189,17 @@ class SpectrumAnalyzerEffect : public LEDStripEffect, virtual public VUMeterEffe
 {
   protected:
 
-    bool      _bShowVU;

```

Comment:
You'll see a lot of simple reordering of member vars to match the inheritance order of constructors and so on